thymio_natives.c

```diff
@@ -545,3 +546,41 @@ void thymio_native_sd_seek(AsebaVMState * vm) {
 
 	vm->variables[status] = sd_user_seek(seek * 2);
 }
+
+AsebaNativeFunctionDescription AsebaNativeDescription_rf_nodeid = {
+	"_rf.nodeid",
+	"Set Wireless Node ID",
+	{
+		{1, "nodeID"},
+		{0,0},
+	}
+};
+
+void set_rf_nodeid(AsebaVMState * vm) {
+	unsigned int nodeid = vm->variables[AsebaNativePopArg(vm)];
+	rf_set_node_id(nodeid);
+	rf_flash_setting();
+}
+
+AsebaNativeFunctionDescription AsebaNativeDescription_soft_reset = {
+	"system.softreset",
+	"Reset state of Thymio as it is when launching VM",
+	{
+		{0,0},
+	}
+};
+
+void soft_reset(AsebaVMState * vm) {
+
+	leds_set_circle(0,0,0,0,0,0,0,0);
+	leds_set_body_rgb(0,0,0);
+	leds_set(LED_SOUND,0);
+	leds_set(LED_RC,0);
+	behavior_start(B_LEDS_ACC);
+	behavior_start(B_LEDS_NTC);
+	behavior_start(B_LEDS_MIC);
+	behavior_start(B_LEDS_PROX);
+	behavior_start(B_SOUND_BUTTON);
+	behavior_start(B_LEDS_RC5);
+	prox_disable_network();
+}
```

Comment:
I don't understand why we need the soft reset function since now we have a proper reset callback from the VM.